gateways/Gateways.md

```diff
@@ -13,18 +13,24 @@ All fjåge Gateway implementations should implement the following classes and me
 - Must return a `null` if the connection to master container fails on the first attempt.
 - May support auto-reconnect, where, once a connection with the master container is established if the connection fails, the Gateway tries to reconnect automatically.
 
-### `getAgentID()` :: Void -> AgentID
+### `authenticate` :: String credentials -> Boolean
+- Tries to authenticate the Gateway connection with the master container.
+- Must send the credential string using command `{"action":"auth","creds":"boo"}`
+- Must wait for a the first `{"auth": true}` or `{"auth": false}` received from the master and return the result.
+- Must default timeout to 1000 millisecond.
```

Comment:
Is 1000 ms a MUST?

Reply:
Good point. Will change to a suggestion.